mrbelvedereci/plan/management/commands/compressplans.py

```diff
@@ -0,0 +1,96 @@
+from itertools import combinations
+
+from django.core.management.base import BaseCommand
+from django.core.management.base import CommandError
+
+from mrbelvedereci.plan.models import Plan
+from mrbelvedereci.plan.models import PlanRepository
+from mrbelvedereci.utils import is_attr_equal
+
+
+class Command(BaseCommand):
+    help = '"Compress" plans by merging redundant plans into a single plan'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--delete',
+            action='store_true',
+            dest='delete',
+            default=False,
+            help='Delete plan instead of deactivating',
+        )
+        parser.add_argument(
+            '--dry-run',
+            action='store_true',
+            dest='dry_run',
+            default=False,
+            help='Do a dry run (do not modify db)',
+        )
+
+    def handle(self, *args, **options):
+        if options['dry_run']:
+            self.stdout.write('DRY RUN - db will not be modified')
+        invalid_plans = set()
+        plan_pks = sorted([plan.pk for plan in Plan.objects.all()])
+        for plan_combo in combinations(plan_pks, 2):
```

Comment:
consider
`for (plan1pk, plan2pk) in combinations(plan_pks, 2)` ?

Reply:
Actually it's useful to leave this as a tuple for the `set` statement on the next line.